docs/1_usage.md

```diff
@@ -2,69 +2,53 @@
 
 Lean how to use the ReVanced CLI.
 
-## ⚡ Setup (optional)
+## ⚡ Setup
 
-1. Make sure your device is connected
+1. Download Java 19: https://jdk.java.net/19/ then unzip it
+2. Download latest release (.jar) of revanced CLI: https://github.com/revanced/revanced-cli/releases
+3. Download latest release (.jar) of revanced patches: https://github.com/revanced/revanced-patches/releases
+4. Download latest release (.apk) of revanced integrations: https://github.com/revanced/revanced-integrations/releases
+5. Download the appropriate version of the Youtube APK: https://www.apkmirror.com/apk/google-inc/youtube/youtube-18-03-36-release/ (replace the version in the URL if necessary; make sure to download the APK and not the APK bundle)
 
-   ```bash
-   adb shell exit
-   ```
-
-   If you plan to use the root variant, check if you have root access
-
-   ```bash
-   adb shell su -c exit
-   ```
-
-2. Copy the ADB device name
-
-   ```bash
-   adb devices
-   ```
+Put all of that in the same directory for easier usage.
 
 ## 🔨 ReVanced CLI Usage
 
+You may need to slightly modify the commands to adapt to your specific versions of Java and/or Revanced.
+
 - ### Show all available options for the ReVanced CLI
 
   ```bash
-  java -jar revanced-cli.jar -h
+  .\openjdk-19.0.2_windows-x64_bin\jdk-19.0.2\bin\java.exe -jar .\revanced-cli-2.20.0-all.jar -h
   ```
 
 - ### List all available patches from supplied patch bundles
 
   ```bash
-  java -jar revanced-cli.jar \
-      -b revanced-patches.jar \
-      -l
+  .\openjdk-19.0.2_windows-x64_bin\jdk-19.0.2\bin\java.exe -jar .\revanced-cli-2.20.0-all.jar -b .\revanced-patches-2.161.1.jar -l --apk '.\com.google.android.youtube_18.03.36-1535632832_minAPI26(arm64-v8a,armeabi-v7a,x86,x86_64)(nodpi)_apkmirror.com.apk' --with-packages --with-versions
   ```
 
-- ### Use the ReVanced CLI without root permissions
+- ### Generate the APK with the patches you want
```

Comment:
The synonym "Generate" has never been used across ReVanced, so it isn't recommended to introduce it here randomly. Stick to already-known words such as "Patch the APK".